ops/provision/modules/k8s/files/prod_issuer.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: cert-manager
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: support@notch8.com
```

Comment:
Not sure what this process looks like, but did we want to switch this from a `notch8.com` email?

Reply:
This email is only used if the renew fails. it should probably be y'all. Updating it is just changing it here and then deploying.

Reply:
👍